### PR TITLE
DAOS-11634 doc: misc doc improvements

### PR DESCRIPTION
**Pull request by Michael-Hennecke**
misc doc improvements

Doc-only: true
Signed-off-by: Michael Hennecke <michael.hennecke@intel.com>

### CODE REVIEW

**Review by kccain — CHANGES_REQUESTED** [review]

File: docs/QSG/tour.md [hunk 1]
```diff
@@ -250,14 +250,14 @@ bring-up DAOS servers and clients.
 	  -----             ----------- ------------
 	  boro-[8,35,52-53] 1           0
 
-	$ dmg pool create --size 10G Pool1
-	$ daos cont create --label Cont1 --type POSIX --oclass RP_3G1 --properties rf:2 Pool1
-	$ daos pool list-cont Pool1
+	$ dmg pool create --size 10G Pool2
```
[the remark this answers is not in the document]
Comment (Michael-Hennecke) [reply docs/QSG/tour.md:253]: lets defer this to a separate PR

File: docs/QSG/tour.md [hunk 2]
```diff
@@ -250,14 +250,14 @@ bring-up DAOS servers and clients.
...
+	$ daos cont create --label Cont1 --type POSIX --oclass RP_3G1 --properties rf:2 Pool2
+	$ daos pool list-cont Pool2
 	UUID                                 Label
 	----                                 -----
 	2649aa0f-3ad7-4943-abf5-4343205a637b Cont1
 
-	$ dmg pool query Pool1
```
[the remark this answers is not in the document]
Comment (Michael-Hennecke) [reply docs/QSG/tour.md:260]: lets defer this to a separate PR

File: docs/QSG/tour.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (Michael-Hennecke) [reply docs/QSG/tour.md:253]: renamed to Cont2 to avoid that ambiguity

[the remark this answers is not in the document]
Comment (saurabhtandan) [reply docs/QSG/tour.md:253]: +1

File: docs/QSG/tour.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (kccain) [reply docs/QSG/tour.md:260]: Yes I think that makes sense.

File: docs/QSG/tour.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (kccain) [reply docs/QSG/tour.md:253]: Sure, sounds good.

**Review by johannlombardi — APPROVED** [review]

**Pull request merged**